docs/resources/artifactory_remote_cargo_repository.md

```diff
@@ -27,4 +27,9 @@ Arguments have a one to one mapping with the [JFrog API](https://www.jfrog.com/c
 * `key` - (Required) The repository identifier. Must be unique system-wide
 * `anonymous_access` - (Required) - Cargo client does not send credentials when performing download and search for crates. Enable this to allow anonymous access to these resources (only), note that this will override the security anonymous access option.
 * `git_registry_url` - (Optional) - This is the index url, expected to be a git repository. for remote artifactory use "arturl/git/repokey.git"
+* `content_synchronisation` - (Optional) Reference [JFROG Smart Remote Repositories](https://www.jfrog.com/confluence/display/JFROG/Smart+Remote+Repositories)
```

Comment:
URL is missing?

Reply:
URL is set to https://www.jfrog.com/confluence/display/JFROG/Smart+Remote+Repositories ... right?